### ISSUE TITLE
Setup assistance

### ISSUE THREAD

**Issue by jdownes79** [oktadeveloper/okta-wordpress-sign-in-widget#29]
I've installed this in a Wordpress site, added a SPA app to Okta, and assigned it to myself.  Neither going to the login page or clicking the App in Okta automatically logs me in.  I get the Okta login page, but it does not log me in.

The plugin in Wordpress says to "Find your Issuer URI in the Admin console under Security -> API"  however when I go to my Okta Admin console Security -> API, I don't see this.  The Okta Issuer URI text box says: "e.g. https://youroktadomain.okta.com/oath2/default" so I added it based on that:  https://myoktadomain.okta.com/oath2/default .   Except I used my actual okta domain.

I get the okta login screen regardless of whether I'm logged into Okta on the same browser.  Trying to log in to the Okta login, I get the error "There was an unexpected internal error.  Please try again."

I am not sure what I am doing wrong.

**Comment by jonascencio** [comment]
Same

**Comment by lmartins** [comment]
Is this being maintained at all?

**Comment by aaronpk** [comment]
The instructions are written for the developer console so try looking there for the issuer URI. I'm honestly not sure if this works with the non-developer account.

This is not an officially supported Okta product, so it's community maintained.

**Comment by lmartins** [comment]
@aaronpk thank you for the feedback and sorry for the frustration. I was really under the impression this was provided by Okta, given that I got here from a post authored by you on their blog. If that isn't the case, my apologies, as my expectations were not in the right place.

I've managed to pull the URL needed for the plugin configuration, but it still throws a blank screen with `AuthApiError`, which I suspect is the same error as the original poster is having.
